examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,88 @@
+"""
+Decoding with :class:`nilearn.decoding.Decoder`: face vs house recognition
+==============================================================
```

Comment:
The line under the title is too short.
```suggestion
=========================================================================
```